versioned_docs/version-v2/introduction/upgrading-to-stencil-three.md

```diff
@@ -8,7 +8,9 @@ contributors:
 
 # Upgrading to Stencil v3.0.0
 
-> Stencil 3.0.0 is pre-release software. These instructions are for users looking to try an early version of the software
+:::caution
+Stencil 3.0.0 is pre-release software. These instructions are for users looking to try an early version of the software
+:::
```

Comment:
This should go away, no?

Reply:
Ooh, yeah. Agreed - I missed that yesterday